### ISSUE TITLE
3D Sound for WebVR - who is developing this? 

### ISSUE THREAD

**Issue by nelabrown** [w3c/webvr-content-workshop#12]
I am a sound designer working with interaction and multi-channel audio in live performances. A few years back I tinkered with Web Audio API and run a few workshops with art & tech lab G.Hack at [Sonar 2014](http://ghack.eecs.qmul.ac.uk/?p=979) and [Music Tech Fest 2014](http://ghack.eecs.qmul.ac.uk/?p=1192) to encourage female hackers to start tinkering with Web Audio :-) 

I recently attended WebVR workshop in London I wanted to find out more about 3D audio for WebVR:
- what is currently available
- are there tools to interface with other 3D audio software tools/ game audio middleware;
- is this a continuation of Web Audio or a completely new strand of development
- who is involved with development (people, companies that sell VR gear etc.)

I have a few projects in the pipeline that I would like to use WebVR for, but with all of them, 3D audio will be the most important component!

**Comment by Utopiah** [comment]
- Etienne Pinchon recently released https://github.com/etiennepinchon/aframe-resonance
- Marko Kungla explored it but I don't know what's the status

**Comment by mkungla** [comment]
Yes indeed you are right @Utopiah and at the moment in progress building kind of DRC for a Web VR., As having also sound engineering background I have been tackling with that subject quite long time already.

Some resources: 
[Web Audio API](https://github.com/WebAudio/web-audio-api)
[MediaStream API](https://www.w3.org/TR/mediacapture-streams/)
[Google's Resonance Audio SDK](https://developers.google.com/resonance-audio/develop/web/getting-started)  
[GoogleChrome/omnitone](https://github.com/GoogleChrome/omnitone)
[Media Capture and Streams specification (aka getUserMedia)](https://github.com/w3c/mediacapture-main)

**Comment by mkungla** [comment]
Some resources: 
[Web Audio API](https://www.w3.org/TR/webaudio/)
[MediaStream API](https://www.w3.org/TR/mediacapture-streams/)
[Google's Resonance Audio SDK](https://developers.google.com/resonance-audio/develop/web/getting-started)

**Comment by Utopiah** [comment]
There is also https://github.com/GoogleChrome/omnitone/wiki/Integration-of-Omnitone-with-AFRAME by https://twitter.com/hochsays

**Comment by delapuente** [comment]
@mkungla are you attending the workshop?

**Comment by Utopiah** [comment]
Yes he is 👍

**Comment by mkungla** [comment]
@delapuente yes I will

**Comment by delapuente** [comment]
Cool. Perhaps this is good for a follow-up session. It should not be only for continuing the discussion on the main topics but also to cover other identified topics. If there is interest on positional audio, would you mind to be the host of the session, @mkungla ?

**Comment by mkungla** [comment]
@delapuente if there is interest, then of course
